mlir/utils/jenkins/Jenkinsfile.miopen-test

```diff
@@ -59,10 +59,11 @@ pipeline {
                     -DBUILD_SHARED_LIBS=OFF
                     -DLLVM_BUILD_LLVM_DYLIB=OFF
                     -DLLVM_ENABLE_TERMINFO=OFF
+                    -DCMAKE_INSTALL_PREFIX=/opt/rocm
                     --log-level=WARNING
                     -DCMAKE_RULE_MESSAGES=OFF
                     '''
-                            sh 'cd build; make -j $(nproc) libMLIRMIOpen'
+                            sh 'cd build; make -j $(nproc) libMLIRMIOpen all install'
```

Comment:
Does this fix the build issue in the jenkins job? I wonder why we need this `all` and `install` targets? It used to be that just building `libMLIRMIOpen` is enough.

Reply:
It seems to be the case that the `miopen-test` image had an older version of the mlir headers. This means that, when we used the `develop` branch, the build would try to link in old headers and fail.

The alternate fix is to use the `mlir` docker image and install dependencies.

Reply:
Thanks!

> The alternate fix is to use the mlir docker image and install dependencies

Way to go!

Reply:
I haven't actually done the alternate fix, but it's on my TODO list and will likely be the next PR

Reply:
I figure we can keep it as is for now and then follow up in a later PR, since fiddling with the containers is a separate issue